includes/evf-core-functions.php

```diff
@@ -1989,3 +1989,221 @@ function evf_cleanup_logs() {
 	}
 }
 add_action( 'everest_forms_cleanup_logs', 'evf_cleanup_logs' );
+
+/**
+ * Process syntaxes in a text.
+ *
+ * @since 1.7.0
+ *
+ * @param string $text Text to be processed.
+ * @param bool   $escape_html Whether to escape all the htmls before processing or not.
+ * @param bool   $trim_trailing_spaces Whether to trim trailing spaces or not.
+ *
+ * @return string Processed text.
+ */
+function evf_process_syntaxes( $text, $escape_html = true, $trim_trailing_spaces = true ) {
+
+	if ( true === $trim_trailing_spaces ) {
+		$text = trim( $text );
+	}
+	if ( true === $escape_html ) {
+		$text = esc_html( $text );
+	}
```

Comment:
Single tab indentation.

Reply:
@coderkoala It IS a single tab indentation. Can you please re-confirm it by looking at it properly again.

Reply:
@krishna-suwal I see, my bad.